packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1082,9 +1088,9 @@ export default class Meetings extends WebexPlugin {
       const activeLocusUrl = [];
 
       if (locusArray?.loci && locusArray.loci.length > 0) {
-        locusArray.loci.forEach((locus) => {
```

Comment:
@mkesavan13  do you think we need just this change as there are no aync calls , to do you need all the changes mentioned in the PR ?

Reply:
@arun3528 - You are right. Removed the unwanted changes. Sent build to customer and tested as well. Kindly review again.